tst/Codelyzer.Analysis.Languages.UnitTests/VisualBasicHandlerTests.cs

```diff
@@ -0,0 +1,524 @@
+using Xunit;
+using Microsoft.CodeAnalysis;
+using Codelyzer.Analysis.Common;
+using Microsoft.CodeAnalysis.VisualBasic;
+using Xunit.Abstractions;
+using Microsoft.Extensions.Logging.Abstractions;
+using Codelyzer.Analysis.VisualBasic;
+using System.Linq;
+
+namespace Codelyzer.Analysis.Languages.UnitTests
+{
+	public class VisualBasicHandlerTests
+	{
+		private AnalyzerConfiguration Configuration;
+		private readonly ITestOutputHelper _output;
+
+		public VisualBasicHandlerTests(ITestOutputHelper output)
+		{
+			_output = output;
+			Configuration = new AnalyzerConfiguration(LanguageOptions.Vb)
+			{
+				ExportSettings =
+				{
+					GenerateJsonOutput = false,
+				},
+
+				MetaDataSettings =
+				{
+					LiteralExpressions = true,
+					MethodInvocations = true,
+					InvocationArguments = true,
+					DeclarationNodes = true,
+					LambdaMethods = true,
+					InterfaceDeclarations = true
+				}
+			};
+		}
```

Comment:
nit: spacing in this file is rendering differently because tabs were used whereas the rest of the codebase uses spaces. Find & Replace in VS can replace the `\t` chars with 4 spaces.